### ISSUE TITLE
`driver.capabilities["browserName"]` now returns "chrome-headless-shell" for `headless` Chrome on Chrome 120, which causes a few issues

### ISSUE THREAD

**Issue by mdmintz** [seleniumbase/SeleniumBase#2343]
### `driver.capabilities["browserName"]` now returns "chrome-headless-shell" for `headless` Chrome on Chrome 120, which causes a few issues.

(Before, it was always returned `"chrome"` for any type of Chrome.)

This is causing `self.is_chromium()` to return `False` on headless Chrome.

It might also be causing other issues to occur.

**Comment by mdmintz** [comment]
This will get fixed in the next release, which will also contain the `selenium` `4.16` upgrade. Currently, I'm waiting to see if other issues such as https://github.com/SeleniumHQ/selenium/issues/13257 will be a problem.

**Comment by mdmintz** [comment]
On `headless` Edge 120, the value of `driver.capabilities["browserName"]` is "edge-headless-shell".

**Comment by mdmintz** [comment]
This was resolved in `4.22.0` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.22.0

Note that although the result of `driver.capabilities["browserName"]` will remain unchanged (that value comes from the browser itself) the methods that use it will change to work via other means.

Eg. `is_chromium()` becomes:

```python
...
    def is_chromium(self):
        """Return True if the browser is Chrome or Edge."""
        self.__check_scope()
        chromium = False
        if (
            "chrome" in self.driver.capabilities
            or "msedge" in self.driver.capabilities
        ):
            chromium = True
        return chromium
```
